comparator/build.gradle

```diff
@@ -0,0 +1,36 @@
+plugins {
+    id 'java'
+    id 'application'
+}
+
+group 'org.mobilitydata'
+version "${System.getenv("versionTag")}"
+
+sourceCompatibility = JavaVersion.VERSION_1_8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile 'com.google.api-client:google-api-client:1.31.2'
+
+    implementation project(':core')
+    implementation 'com.google.guava:guava:29.0-jre'
+    implementation 'com.google.flogger:flogger:0.5.1'
+    implementation 'com.google.flogger:flogger-system-backend:0.5.1'
+    implementation 'com.beust:jcommander:1.48'
+    implementation 'com.google.code.gson:gson:2.8.6'
+    testImplementation "com.google.truth:truth:1.0.1"
+    testCompile group: 'junit', name: 'junit', version: '4.12'
+}
+
+jar {
+    // Add the manifest within the JAR, using comparator as the title
+    manifest {
+        attributes('Implementation-Title': 'comparator',
+                'Implementation-Version': project.version,
+                'Main-Class': 'org.mobilitydata.gtfsvalidator.outputcomparator.cli.Main',
+                'Multi-Release': 'true')
+    }
```

Comment:
You'll also need the `shadowJar` section to produce the shaded JAR with all the dependencies for this to run on CI via `java -jar`.